pkg/commands/tls/custom/certificate/create.go

```diff
@@ -17,7 +17,7 @@ func NewCreateCommand(parent argparser.Registerer, g *global.Data) *CreateComman
 	c.Globals = g
 
 	// Required.
-	c.CmdClause.Flag("cert-blob", "The PEM-formatted certificate blob").Required().StringVar(&c.certBlob)
+	c.CmdClause.Flag("cert-blob", "The PEM-formatted certificate blob. Can be passed as file path or content, e.g. \"$(< certificate.crt\" or --cert-blob=path/to/certificate.crt").Action(c.certBlob.Set).Required().StringVar(&c.certBlob.Value)
```

Comment:
Slight tweak to the description, but importantly I've removed the `.Value` as I think the type should be a `string` and not an `argparser.OptionalString` (because the flag is marked as required).

```suggestion
	c.CmdClause.Flag("cert-blob", "The PEM-formatted certificate blob. Can be passed as file path (e.g. \"path/to/certificate.crt\") or content (e.g. \"$(< certificate.crt)\")").Action(c.certBlob.Set).Required().StringVar(&c.certBlob)
```